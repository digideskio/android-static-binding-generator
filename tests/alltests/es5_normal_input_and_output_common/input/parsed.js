(function () {
    var MyButton = android.widget.Button2222.extend("NormalClassName", {
        onClick222222: function () {
        },
        onClick222222222222: function () {
        }
    });
})();