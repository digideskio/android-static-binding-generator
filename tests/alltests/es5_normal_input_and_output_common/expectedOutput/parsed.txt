android.widget.Button2222*_fparsed_l2_c46__NormalClassName*onClick222222,onClick222222222222*
